### PR TITLE
Added semantic refs to classes 

### LINKED ISSUE #204: Write a parser to fetch the semantic references of simple enough classes
### I'm submitting a 
- [ ] bug report.
- [x] feature request.

### Current Behaviour:

This is related to the discussion we had here https://docs.google.com/document/d/1J9PlcXUfu76lruJCb_1vKcQauL-J6H2wxZxqzdZO9dQ/edit

### Expected Behaviour:

Every class/property needs to have a semantic reference. If it's a complex object then define the object in vocab itself and provide semantic references to its properties else directly provide a semantic reference.
Either way, semantic references need to be there.

It would be better if you write a parser that'll try to find semantic references for properties from `Schema.org` and similar sources.

### Do you want to work on this issue?
NO

### PR DESCRIPTION
**Pull request by vaibhavchellani**
Fixes #204

### Checklist
- [ X] My branch is up-to-date with upstream/develop branch.
- [X ] Everything works and tested for Python 3.5.2 and above.

### Description

Added semantic refs to classes , working on the external parser , its work in progress , please suggest changes to this .

### CODE REVIEW

**Comment by Mec-iS** [comment]
* Code has to be PEP-8 compliant with docstring for every method.
* Commit messages are too vague and unprecise
* All my annotations on [this comment](https://github.com/HTTP-APIs/python-hydra-agent/pull/23#issuecomment-399654218) apply also to this. In particular I don't see coding cycles to be fast enough to accomplish the second deadline at the moment.

File: hydrus/samples/hydra_doc_sample.py [hunk 1]
```diff
@@ -107,7 +107,7 @@
                 },
                 {
                     "@type": "SupportedProperty",
-                    "property": "vocab#/definitions/Category",
+                    "property": "vocabCategory",
```
Comment (xadahiya) [inline hydrus/samples/hydra_doc_sample.py]: Shouldn't this be `vocab:Category` ?

File: hydrus/parser/openapi_parser.py [hunk 2]
```diff
@@ -37,21 +37,113 @@ def check_if_collection(schema_block: Dict[str, Any]) -> str:
...
 
-def get_class_details(class_location: List[str], doc: Dict["str", Any], classAndClassDefinition: Dict["str",HydraClass],
+def parse_prop(
+        prop:str,
+        properties:Dict["str",Any],
+        definitionSet: Set["str"],
+        doc: Dict["str",Any],
+        classAndClassDefinition:Dict["str",HydraClass]) -> str:
+    """
+
```
Comment (xadahiya) [inline hydrus/parser/openapi_parser.py]: Some Docstrings have `descriptions` but lack `params` other have `params` but lack proper `descriptions`.
Fix this

**Comment by Mec-iS** [comment]
@vaibhavchellani I still don't see docstrings in the methods. As instructed different times, the code has to be compliant with PEP-8.

**Comment by xadahiya** [comment]
Using placeholders at the moment.

**Pull request closed without merging**